### ISSUE TITLE
ApproxMRRLoss

### ISSUE THREAD

**Issue by celsofranssa** [rjagerman/pytorchltr#33]
It would be great if you also provide the [AppoxMRR](https://www.tensorflow.org/ranking/api_docs/python/tfr/keras/losses/ApproxMRRLoss) loss.
